fastats/core/ast_transforms/processor.py

```diff
@@ -22,55 +23,81 @@ def __init__(self, top_level_func, overrides, replaced, new_funcs=None):
         self._debug = self._overrides.get('debug')
 
     def process(self):
-        source = inspect.getsource(self.top_level_func)
-        tree = ast.parse(source)
+        try:
+            source = inspect.getsource(self.top_level_func)
+            tree = ast.parse(source)
+        except IndentationError:
+            data = uncompile(self.top_level_func.__code__)
+            tree = parse_snippet(*data)
+
         globs = self.top_level_func.__globals__
+        globs['jit'] = jit
```

Comment:
This deserves a comment

Reply:
Adding it to the globals dict means that an explicit import isn't required for dynamically generated code. Can add it as a comment, i just thought it was obvious.

Reply:
Nothing's obvious when you're doing AST rewrites ;)